.github/workflows/release-workflow.yml

```diff
@@ -15,9 +15,9 @@ jobs:
         with:
           java-version: '11'
           distribution: 'temurin'
-          cache: 'sbt'
+
       - name: Build tarball
-        run: sbt createTarball
+        run: build/sbt createTarball
```

Comment:
why is this change showing up?

Reply:
> why is this change showing up?

I think its due to how merging/rebasing of main. I'll try to figure out how to fix this up.